test/unit/api/api-llm.test.js

```diff
@@ -58,4 +58,31 @@ tap.test('Agent API LLM methods', (t) => {
       t.end()
     })
   })
+
+  t.test('getLlmMessageIds is no-op when ai_monitoring is disabled', async (t) => {
```

Comment:
not in this PR but we probably want to add more "unit" tests here around additional logic of these API methods.  I know versioned tests execute happy path but not cases like: not in tx context for both, returning empty on message ids

Reply:
I think that's why codecov is failing but might be wrong